bom-weekly/pom.xml

```
@@ -299,6 +299,11 @@
                 <artifactId>credentials-binding</artifactId>
                 <version>523.vd859a_4b_122e6</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
```

Incorrect group ID:

```suggestion
                <groupId>io.jenkins.plugins</groupId>
```

After applying this suggestion, please test locally with `PLUGINS=custom-folder-icon bash local-test.sh` and apply any other necessary fixes.

---

Seems like the heat yesterday was really getting into my head if even c&p fails me 🥵 Fixed it now. 
I ran the tests locally without any further issues.

---

I hear ya 🥵

We are all human! Thanks for contributing to Jenkins and BOM 👏